### PR TITLE
tests: add seed files for Rails.env.development

### PR DESCRIPTION
**Pull request by KellyAH**
Paired with Lailson to create seed files that can be run on local by executing `rails db:seed`.
Running it multiple times will create duplicate data with the same value but different ids.

To create tables containing ONLY the seed data do `rails db:reset`. 
WARN: this will wipe out existing data in your local DB.

You may need to restart the rails server for the seed data to display on the FE.
Seed data will make the FE look like this:

Screen Shot 2021-01-13 at 6 13 50 PM

# reference docs
http://blog.magmalabs.io/2019/11/25/best-practices-using-rails-seeds.html

### CODE REVIEW

File: app/models/task.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Task < ApplicationRecord
+  #TODO: store priority as strings instead of integers e.g. "high" instead of 1 to avoid issues
```
Comment (NOI-lgtm) [inline app/models/task.rb]: I think we can remove this comment. Converting from integer to string is done in the other pull request already. What you think :)?

Comment (KellyAH) [reply app/models/task.rb]: yeah lemme remove it

**Review by NOI-lgtm — APPROVED** [review]
👍

**Review by NOI-lgtm — APPROVED** [review]
👍 💯 LGTM

**Pull request merged**